### DISCUSSION TITLE
版本问题

### DISCUSSION THREAD

**Discussion by NingLiu1998** [Hoshikawa-Kaguya/Sora#85] [Q&A]
报错如图：
![image](https://github.com/Hoshikawa-Kaguya/Sora/assets/109678937/d96739fd-baa1-4582-9908-2dbfd6ebac48)

嘤嘤嘤，俺是7.0强行用的，啥时候支持7.0鸭

**Comment by Yukari316** [comment]
JSON转换错误，检查你的onebot端上报是不是设置成了`string`格式，需要改成`array`格式
以及就是，.net高版本是能够兼容低版本的包的
还有就是，多读下文档，这个文档写了的

**Reply by NingLiu1998** [reply]
嘶，有这样的设置吗，我去康康，谢谢
